### PR TITLE
Linux: aarch64/ppc64le, Apple: aarch64

### PR DESCRIPTION
**Pull request by ax3l**
Enable ARM and PowerPC builds.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged) - not needed here, we just add more build targets
* [ ] Reset the build number to `0` (if the version changed) - not needed here, we just add more build targets
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Review by cemitch99 — APPROVED** [review]
Looks ok to me.

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -33,16 +33,32 @@ cmake \
...
+    # skip the pyAMReX tests to save CI time
+    EXCLUSION_REGEX="AMReX"
+
+    # macOS x86_64 pypy: import issue with matplotlib
+    #   AttributeError: module 'threading' has no attribute 'get_native_id'
+    # https://foss.heptapod.net/pypy/pypy/-/issues/3764
+    if [[ "${target_platform}" == "osx-64" ]]; then
+        IS_PYPY=$(${PYTHON} -c "import platform; print(int(platform.python_implementation() == 'PyPy'))")
+        if [[ ${IS_PYPY} == "1" ]]; then
+            EXCLUSION_REGEX="(AMReX|plot|pytest|iotalattice)"
```
Comment (ax3l) [inline recipe/build.sh:56]: Maybe we can keep `iotalattice` (hang/timeout with pypy) in after #8 is merged

**Comment by ax3l** [comment]
@conda-forge-admin, please rerender

**Pull request merged**